wp-content/plugins/hurumap/index.php

```diff
@@ -330,14 +330,25 @@ function hurumap_data_blocks_register()
         register_block_type('hurumap-data/featured-data', array(
             'editor_script' => 'hurumap-data-blocks-script',
         ));
-
+        
         register_block_type('hurumap/card-block', array(
             'editor_script' => 'hurumap-data-blocks-script',
         ));
-
+        
         register_block_type('hurumap/indicator-block', array(
             'editor_script' => 'hurumap-data-blocks-script',
         ));
+
+        register_block_type('hurumap/section-chart', array(
+            'editor_script' => 'hurumap-data-blocks-script',
+        ));
+        register_block_type('hurumap/charts-row-block', array(
+            'editor_script' => 'hurumap-data-blocks-script',
+        ));
+
+        register_block_type('hurumap/section-block', array(
+            'editor_script' => 'hurumap-data-blocks-script',
+        ));
```

Comment:
A bit documentation on these: Section Block -> Charts Row Block -> Section Chart? If so, may be `section-block`, `section-row-block` & `section-row-chart-block` would make this relation clear... Not sure though.

Reply:
@kilemensi yes, I will rename them